src/main/java/seedu/parser/Parser.java

```diff
@@ -55,53 +56,59 @@ public static Command parse(String userInput) throws EscException {
-            return prepareAddSubject(arguments);
+            case AddSubjectCommand.COMMAND_WORD:
+                return prepareAddSubject(arguments);
 
-        case AddCardCommand.COMMAND_WORD:
-            return prepareAddCard(arguments);
+            case AddCardCommand.COMMAND_WORD:
+                return prepareAddCard(arguments);
 
-        case DeleteSubjectCommand.COMMAND_WORD:
-            return prepareDeleteSubject(arguments);
+            case EditCardCommand.COMMAND_WORD:
+                return prepareEditCard(arguments);
 
-        case DeleteCardCommand.COMMAND_WORD:
-            return prepareDeleteCard(arguments);
+            case DeleteSubjectCommand.COMMAND_WORD:
+                return prepareDeleteSubject(arguments);
 
-        case ListSubjectCommand.COMMAND_WORD:
-            return new ListSubjectCommand();
+            case DeleteCardCommand.COMMAND_WORD:
+                return prepareDeleteCard(arguments);
 
-        case ListCardCommand.COMMAND_WORD:
-            return prepareListCard(arguments);
+            case ListSubjectCommand.COMMAND_WORD:
+                return new ListSubjectCommand();
 
-        case QuizCommand.COMMAND_WORD:
-            return prepareQuiz(arguments);
+            case ListCardCommand.COMMAND_WORD:
+                return prepareListCard(arguments);
 
-        case ScoreCommand.COMMAND_WORD:
-            return prepareScore(arguments);
+            case QuizCommand.COMMAND_WORD:
+                return prepareQuiz(arguments);
 
-        case AddEventCommand.COMMAND_WORD:
-            return prepareAddEvent(arguments);
+            case ScoreCommand.COMMAND_WORD:
+                return prepareScore(arguments);
 
-        case DeleteEventCommand.COMMAND_WORD:
-            return prepareDeleteEvent(arguments);
+            case AddEventCommand.COMMAND_WORD:
+                return prepareAddEvent(arguments);
 
-        case ListEventCommand.COMMAND_WORD:
-            return new ListEventCommand();
+            case DeleteEventCommand.COMMAND_WORD:
+                return prepareDeleteEvent(arguments);
 
-        case ShowUpcomingCommand.COMMAND_WORD:
-            return prepareShowUpcoming(arguments);
+            case ListEventCommand.COMMAND_WORD:
+                return new ListEventCommand();
 
-        case HelpCommand.COMMAND_WORD:
-            return new HelpCommand();
+            case ShowUpcomingCommand.COMMAND_WORD:
+                return prepareShowUpcoming(arguments);
 
-        case ExitCommand.COMMAND_WORD:
-            return new ExitCommand();
+            case HelpCommand.COMMAND_WORD:
+                return new HelpCommand();
 
-        default:
-            throw new EscException(INCORRECT_COMMAND + HelpCommand.MESSAGE_USAGE);
+            case ExitCommand.COMMAND_WORD:
+                return new ExitCommand();
+
+            default:
+                throw new EscException(INCORRECT_COMMAND + HelpCommand.MESSAGE_USAGE);
```

Comment:
According to the convention set in lecture, these should not have the extra tab spacing. The default & set cases should be in line with the switch command. Pls update!

(From the textbook)
![image](https://user-images.githubusercontent.com/15946120/78969733-45f19000-7b3a-11ea-9c40-ac90b5a007f1.png)